x-pack/plugins/security_solution/cypress/integration/detection_rules/bulk_edit_rules.spec.ts

```diff
@@ -123,6 +131,61 @@ describe('Detection rules, bulk edit', () => {
     hasIndexPatterns([...DEFAULT_INDEX_PATTERNS, CUSTOM_INDEX_PATTERN_1].join(''));
   });
 
+  it('should warn before add/delete/overwrite index patterns in rules when rule with dataview id is selected', () => {
+    cy.log('Creates custom rule with dataViewId');
+    createCustomRule(customRuleWithDataViewId, 'rule with dataview id');
```

Comment:
I tested this on my local. The output of the method is a regular rule with index patterns, not a data view.

<img width="1266" alt="Screenshot 2022-05-20 at 17 39 32" src="https://user-images.githubusercontent.com/17427073/169573862-6d0e5b2f-84c7-4127-bdc0-81454973c63d.png">

<img width="388" alt="Screenshot 2022-05-20 at 18 44 49" src="https://user-images.githubusercontent.com/17427073/169574372-914b5895-36d8-41b8-8982-eb88ae534eba.png">


	.